tests/transferChatResolve.test.ts

```diff
@@ -143,7 +152,7 @@ describe('transferChatResolve', () => {
     };
 
     await Promise.all(
-      [{}, event1, event2, event3, event4].map(event =>
+      [event1, event2, event3, event4].map((event) =>
```

Comment:
This is removing one test case?

Reply:
@GPaoloni , I will refactor it. The problem here is passing an empty `{}` will throw an error due to Typescript is expecting `request` within the empty object. I will refactor it to be so.